ispcrt/CMakeLists.txt

```diff
@@ -189,10 +191,13 @@ if (WIN32)
     set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 endif()
 build_ispcrt(SHARED ${PROJECT_NAME})
-if (WIN32)
-    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS OFF)
+
+if (ISPCRT_BUILD_STATIC)
+  if (WIN32)
+      set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS OFF)
+  endif()
```

Comment:
It seems that `CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS` relates to building of shared library: we set it `ON` on line 191, build the shared library, and should restore it back to `OFF` after `build_ispcrt(SHARED ${PROJECT_NAME})`.
Please move it from `if (ISPCRT_BUILD_STATIC)` condition.

Reply:
OK, fixed.